.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+on:
+  pull_request:
```

Comment:
> It might be good to have the same check when creating a commit as well, would you mind adding that?

@shelegdmitriy as you can see, this workflow will be triggered on pull requests as well

Reply:
Oh I see, great!